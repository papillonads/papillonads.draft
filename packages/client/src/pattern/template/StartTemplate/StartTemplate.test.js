import React from 'react'
import startTemplateDefault from './StartTemplate.render'

jest.mock('react-router-dom', () => {
  return {
    Link: () => <div />,
  }
})

describe('<StartTemplate />', () => {
  describe('Snaphot', () => {
    test('must match startTemplateDefault', () => {
      expect(global.renderToJSON(startTemplateDefault)).toMatchSnapshot()
    })
  })
})
