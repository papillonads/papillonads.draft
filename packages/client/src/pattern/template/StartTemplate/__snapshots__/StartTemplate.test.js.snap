// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StartTemplate /> Snaphot must match startTemplateDefault 1`] = `null`;
